src/routes/design-tokens/overview/Overview.css

```diff
@@ -1,3 +1,6 @@
 .overview-page goa-container {
   max-width: 72ch;
 }
+.overview-page code {
```

Comment:
Instead let's create an app level css class `inline-code`, since I can see this being used elsewhere.